R/sun-methods.R

```diff
@@ -76,13 +109,31 @@ setMethod("crepuscule", signature(crds="SpatialPoints", dateTime="POSIXct"),
                          direction=direction, POSIXct.out=POSIXct.out)
           })
 
-###_ + sunriset methods
-if (!isGeneric("sunriset")) {
-    setGeneric("sunriset", function(crds, dateTime, ...) {
-        standardGeneric("sunriset")
-    })
-}
+#' @title Calculate sunrise/sunset
+#' @description Calculates sunrise or sunset at a specific geographical location and time.
+#' depending on the "direction" parameter. Methods are available
+#' for differet object types with geographical coordinates, including:
+#' * "sf": an object of class "sf".
+#' * "matrix": a matrix of coordinates.
+#' * "SpatialPoints": an object of class "SpatialPoints".
+#'
+#' @param crds This represents geographical coordinates. It can be an object of
```

Comment:
- remove `This represents`
- for matrix you need a bit more info, should matrix be named? should lat or lon come first?